src/main/java/com/grapeup/webapp/testWebApp/repositories/SessionRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.grapeup.webapp.testWebApp.repositories;
+
+import com.grapeup.webapp.testWebApp.models.Session;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface SessionRepository extends JpaRepository<Session, Long> {
```

Comment:
I suppose you don't have a reason to use `JpaRepository` over `CrudRepository` https://www.tutorialspoint.com/difference-between-crudrepository-and-jparepository-in-java